app/views/supervisor/dashboard/index.html.slim

```diff
@@ -0,0 +1,10 @@
+h1 = "Hi #{current_user.name}!"
+h2 This is your brandnew Supervisor Dashboard - with love from Team Cheesy
+
+hr
+
+- for t in supervised_teams.each do
```

Comment:
I am not sure about the slim format. Did you happen to check if they both list all @supervised_teams ?